CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contributing to Stryker Dashboard
+
+This is the contribution guide for Stryker Dashboard. Great to have you here! Here are a few ways you can help make this project better.
+
+## Creating issues
+
+Do you have an idea for a feature or have you found a bug? Please create an issue so we can talk about it!
+
+## Adding new features
+
+New features are welcome! Either as requests or proposals.
+
+1. Please create an issue first, so we know what to expect from you.
+2. Create a fork on your github account.
+4. Please create or edit unit tests or integration tests.
+5. Run the tests and verify they pass.
+6. Update documentation.
+
+## Maintainers
+
+### Using Milestones
+
+Milestones are used to group issues and pull requests together to provide a clear overview of the project's progress. When submitting or reviewing issues, please consider the following guidelines related to milestones:
+
+1. Assigning Milestones: When creating or reviewing an issue, consider whether it aligns with any existing milestones. If it does, please assign the appropriate milestone to the issue to ensure it is properly tracked.
+
+2. Creating New Milestones: If you believe that a new milestone is necessary to better organize and track the project's progress, please consult with the project maintainers before creating a new milestone.
+
+3. Tracking Progress: By utilizing milestones, we can collectively track progress and better understand the project's development roadmap.
+
+4. Prioritizing Issues: Milestones can also help in prioritizing issues and feature requests, providing a clearer picture of what needs to be addressed in each development cycle.
+
+By adhering to these guidelines, we can ensure that the use of milestones enhances the clarity and organization of our project's development process. Once a milestone is complete, we can release the next version.
+
+### Naming conventions
+
+When merging pull requests or creating commits, please conform to the angular commit message style, so our changelog will be updated. Namely in the form `<type>(<scope>): <subject>\n\n[body]`
+
+* Type: feat, fix, docs, style, refactor, test, chore.
+* Scope: the file or group of files (not a strict right or wrong)
+* Subject and body: present tense (~~changed~~change, ~~added~~add) and include motivation and contrasts with previous behavior
+
+## Community
+
+Do you want to help? Great! These are a few things you can do:
+
+* Evangelize mutation testing. Mutation testing is still relatively new, especially in .NET Core. Please help us get the word out there!
+* Share your stories in blog posts an on social media. Please inform us about it! Did you use Stryker? Your feedback is very valuable to us. Good and bad! Please contact us and let us know what you think
```

Comment:
Maybe add the Stryker social media links to this file so people can easily tag or reach out to us.